### PR TITLE
Chinese translation v2.0

### PR DESCRIPTION
**Pull request by liyishuai**

### CODE REVIEW

**Comment by CoralineAda** [comment]
Thanks @liyishuai! I'll put out a call for other native speakers to review the translation. Your effort is greatly appreciated! 🖤

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/cn"]
++++
+
+# 参与者公约
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md]: 个人认为"Contributor"还是翻译为“贡献者”
```suggestion
# 贡献者公约
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+身为项目成员、贡献者、负责人，我们保证参与此社区的每个人都不受骚扰，不论其年龄、体型、身体条件、民族、性征、性别认同与表现、经验水平、教育程度、社会地位、国籍、相貌、种族、宗教信仰及性取向如何。
+
+我们承诺致力于建设开放、友善、多元、包容、健康的社区环境。
+
+## 我们的准则
+
+有助于促进本社区积极环境的行为包括但不限于：
+
+* 与人为善、推己及人
+* 尊重不同的主张、观点和经验
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md]:
```suggestion
* 尊重不同的主张、观点和经历
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+* 积极提出、耐心接受有益批评
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md:20]:
```suggestion
* 积极提出、耐心接受建设性意见
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+* 面对过失，承担责任、认真道歉、从中学习
+* 关注社区共同诉求，而非一己私利
+
+不当行为包括但不限于：
+
+* 发布与性有关的言论或图像，以及任何形式的献殷勤或勾引
+* 挑衅行为、侮辱或贬损的言论、人身及政治攻击
+* 公开或私下骚扰
+* 未获明确授权擅自发布他人的资料，如地址、电子邮箱等
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md:29]:
```suggestion
* 公开他人的隐私信息，例如未经其许可公开其物理地址或邮箱地址
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+* 其他有理由认定为违反职业操守的不当行为
+
+## 落实之义务
+
+社区负责人有责任诠释何谓“妥当行为”，并据此准则，妥善公正地认定与处置不当、威胁、冒犯及有害的行为。
+
+社区负责人有权利和义务删除、编辑、拒绝违背本标准的评论（comment）、提交（commit）、代码、维基（wiki）编辑、问题（issue）等贡献。如有必要，需告知采取措施之理由。
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md]:
```suggestion
社区负责人有权利和义务删除、编辑、拒绝违背本公约的评论（comment）、提交（commit）、代码、维基（wiki）编辑、问题（issue）等贡献。如有必要，需告知采取措施之理由。
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+## 适用范围
+
+此行为标准适用于本社区全部场合，以及在其他场合代表本社区的个人。
+
+代表本社区的情形包括但不限于：使用官方电子邮件与社交平台、作为指定代表参与在线或线下活动。
+
+## 贯彻落实
+
+如遇滥用、骚扰等不当行为，请通过［在此输入联系方式］向纪律检查委员举报。
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md:46]: 这里原文是"community leaders"，译为“纪律检查委员”不妥。下同。

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+纪委将审议并调查全部投诉。
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md]:
```suggestion
纪委将迅速审议并调查全部投诉。
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+**社区影响**：用语不当、举止不符合职业道德或不受社区欢迎。
+
+**处理意见**：由社区负责人予以非公开的书面警告，阐明违纪事由、解释举止如何不妥。或将要求公开道歉。
+
+### 二、警告
+
+**社区影响**：一起或多起事件中的违纪行为。
+
+**处理意见**：警告继续违纪之后果、违纪者在特定时间内禁止与当事人往来、不得擅自与社区执法者往来，禁令涵盖社区内外、社交网络在内的一切联络。如有违反，可致封禁乃至开除。
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md:65]:
```suggestion
**处理意见**：警告继续违纪之后果、违纪者在特定时间内禁止与当事人往来、不得擅自与社区执法者往来，禁令涵盖社区内以及社区外包括社交媒体在内的一切联络。如有违反，可致封禁乃至开除。
```

File: content/version/2/0/code_of_conduct.zh-cn.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+**处理意见**：违纪者在特定时间内禁止与社区的任何往来或公开联络，禁止任何与当事人公开或私下往来，不得擅自与社区执法者往来。如有违反，可致开除。
+
+### 四、开除
+
+**社区影响**：典型违纪行为，例如屡教不改、骚扰某个人、敌对或贬低某个群体。
+
+**处理意见**：无限期禁止违纪者与项目社区的一切公开往来。
+
+## 来源
```
Comment (kj415j45) [inline content/version/2/0/code_of_conduct.zh-cn.md:79]: 需同步上游更新

**Review by kj415j45 — CHANGES_REQUESTED** [review]

File: content/version/2/0/code_of_conduct.zh-cn.md (hunk 3 above)
Comment (liyishuai) [reply content/version/2/0/code_of_conduct.zh-cn.md:20]: “建设性”系直译，其语义不清，故主张本土化为“有益”。

File: content/version/2/0/code_of_conduct.zh-cn.md (hunk 4 above)
Comment (liyishuai) [reply content/version/2/0/code_of_conduct.zh-cn.md:29]: “Physical”直译为“物理”欠妥。

File: content/version/2/0/code_of_conduct.zh-cn.md (hunk 6 above)
Comment (liyishuai) [reply content/version/2/0/code_of_conduct.zh-cn.md:46]: 此处本土化确实激进，但不妨取其字面意思，负责调查的community leader自然可称为纪委。

File: content/version/2/0/code_of_conduct.zh-cn.md (hunk 8 above)
Comment (liyishuai) [reply content/version/2/0/code_of_conduct.zh-cn.md:65]: “社区”一词重复出现，故主张合并为“内外”。

**Review by liyishuai — COMMENTED** [review]
Thanks for your review! I've merged some suggestions while others are still under discussion.

**Review by ghost — APPROVED** [review]

**Comment by liyishuai** [comment]
@CoralineAda Ready to merge.

**Comment by CoralineAda** [comment]
Thank you @ liyishuai!

**Review by invictus80 — APPROVED** [review]

**Review by Jacobwer — COMMENTED** [review]
Gracias

**Review by Jacobwer — COMMENTED** [review]
Perfecto muchas gracias

**Pull request merged**